docs/_source/practical_guides/assign_records.md

```diff
@@ -15,162 +12,63 @@ You will need to decide the level of overlap before creating or pushing a datase
-users = [user for user in users if user["role"]=="annotator"]
+records = [...]
 ```
 
-2. Get a list of the records that each will annotate:
+2. Assign the annotations using the `assign_records` function:
 
 ```python
-import random
-from collections import defaultdict
-
-records = [...] # A Python list containing all the records in your dataset
-
-# Optional: shuffle the records to get a random assignment
-random.shuffle(records)
-
-# Build a dictionary where the key is the username and the value is the list of records assigned to them
-assignments = defaultdict(list)
-
-# Divide your records into chunks of the same length as the user list and make the assignments
-# You will need to follow the instructions to create and push a dataset for each of the key-value pairs in this dictionary
-n = len(users)
-chunked_records = [records[i:i + n] for i in range(0, len(records), n)]
-for chunk in chunked_records:
-    for idx, record in enumerate(chunk):
-        assignments[users[idx].username].append(record)
-```
-
-3. Loop through the dictionary of assignments to create one dataset per user:
-
-```{note}
-If you haven't done so already, decide on the settings of the project (the `fields`, `questions` and `guidelines`) as detailed in the [Create a Feedback Dataset guide](/practical_guides/create_dataset.md#feedback-dataset) and set those as variables.
+from argilla.client.feedback.utils import assign_records
+assignments = assign_records(users, records, 1, True)
+
+# assign_records(users, records, overlap, shuffle): a dictionary with the assignments
+    # users: a list of users or a dictionary with groups of users
+    # records: a list of records
+    # overlap: the number of annotations per record
+    # shuffle: whether to shuffle the records before assigning them
 ```
 
-::::{tab-set}
+3. Assign the records by pushing a dataset to their personal workspace using `assign_workspaces`:
 
-:::{tab-item} Argilla 1.14.0 or higher
 ```python
-fields = [...]
-questions = [...]
-guidelines = "..."
-
-for username, records in assignments.items():
-    # Check that the user has a personal workspace and create it if not
-    try:
-        workspace = rg.Workspace.from_name(username)
-    except:
-        workspace = rg.Workspace.create(username)
-        user = rg.User.from_name(username)
-        workspace.add_user(user.id)
-
-    # Create a dataset with their assignment and push it to their workspace
-    dataset = rg.FeedbackDataset(fields=fields, questions=questions, guidelines=guidelines)
-    dataset.add_records(records)
-    remote_dataset = dataset.push_to_argilla(name="my_dataset", workspace=workspace.name)
-```
-:::
+from argilla.client.feedback.utils import assign_workspaces
+wk_assignments = assign_workspaces(assignments, "individual")
 
-:::{tab-item} Lower than Argilla 1.14.0
-```python
-fields = [...]
-questions = [...]
-guidelines = "..."
+# assign_workspace(assignments, workspace_type): to check the workspaces
```

Comment:
here you can also use explicit variable name 

```python
assign_workspace(
    assignments=assignments, 
    workspace_type="group"
)
```
you can add additional context about the workspace types outside of the code block